### PR TITLE
chore: Add Accelerator labels

### PR DESCRIPTION
**Pull request by engedaam**
Fixes # 

**Description**
- Added new accelerator labels that support aws neuron

**How was this change tested?**
- Manual testing 

**Does this change impact docs?**
- [x] Yes, PR includes docs updates
- [ ] Yes, issue opened: # 
- [ ] No

**Release Note**

```release-note

```

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: pkg/apis/v1alpha1/register.go [hunk 1]
```diff
@@ -75,6 +75,9 @@ var (
 	LabelInstanceGPUCount                     = LabelDomain + "/instance-gpu-count"
 	LabelInstanceGPUMemory                    = LabelDomain + "/instance-gpu-memory"
 	LabelInstanceAMIID                        = LabelDomain + "/instance-ami-id"
+	LabelInstanceAcceleratorName              = LabelDomain + "/instance-accelerator-name"
```
Comment (ellistarn) [inline pkg/apis/v1alpha1/register.go:78]: Do we still need a concept of `gpu` now that we have the concept of `accelerator`?

Comment (ellistarn) [reply pkg/apis/v1alpha1/register.go:78]: Is `accelerator-memory` something we should expose?

Comment (ellistarn) [reply pkg/apis/v1alpha1/register.go:78]: One option is to continue to support gpu labels, but remove them from the docs.

Comment (engedaam) [reply pkg/apis/v1alpha1/register.go]: > Do we still need a concept of `gpu` now that we have the concept of `accelerator`?

Leaving the gpu label until v1beta1 for depracation

**Review by jonathan-innis — APPROVED** [review]
LGTM 🚀

**Pull request merged**